examples/vars.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+# BBB
+bbb_state: present # For updating BigBlueButton with this role use 'latest'
+#bbb_hostname: <your host fqdn> # Uncomment and fill if bbb fqdn is different than default {{ ansible_fqdn }}
+
+# LETSENCRYPT
+#bbb_letsencrypt_email: "<your email>" # Uncomment and fill if using letsencrypt for TLS certificates
+
+# COTURN
+#bbb_coturn_secret: <secret> # Uncomment, generate and fill using `openssl rand -hex 16`, when using coturn
+
+# GREENLIGHT
+#bbb_greenlight_secret: <secret> # Uncomment, generate and fill using `openssl rand -hex 64`, when using greenlight
+#bbb_greenlight_db_password: <secret> # Uncomment, generate and fill using `openssl rand -hex 16`, when using greenlight
+#bbb_greenlight_users: # Uncomment and fill if using greenlight, to create users
+#  - { name: '', email: '', password: '', type: 'user' }
+#  - { name: '', email: '', password: '', type: 'admin' }
+
+# FREESWITCH
+bbb_freeswitch_socket_password: "<secret>" # Generate and fill using `pwgen -s 16 1`
```

Comment:
I would suggest to write a hint to use ansible-vault for secrets.

Reply:
Do you think the hint I added is enough, or a more detailed explanation would be preferable? If so, where should it go?

Reply:
This looks fine, i my opinion :)